src/components/ContactsList/Contacts/ContactName.jsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const ContactName = ({ firstname, lastname }) => (
-  <div className="u-ellipsis">
-    <span className="contact-firstname">{firstname}</span>
-    &nbsp;
-    <span className="contact-lastname">{lastname}</span>
-  </div>
-)
+const ContactName = ({ displayName, lastname }) => {
+  const namesToDisplay = (displayName && displayName.split(' ')) || []
+  return (
+    <div className="u-ellipsis">
+      {namesToDisplay.map((name, key) => (
+        <span key={`display-${key}`}>
+          <span className={name === lastname ? 'contact-lastname' : 'u-ml-1'}>
```

Comment:
Can you share a screenshot of the list with : one contact having a lastname and a name, and one only with something to display but not a lastname ? I think the one without a lastname will start with a left margin and this is not what we want, right? 

Also, I think we can remove the contact-lastname class and use u-bold or something ;)

Reply:
Margin are now fixed.
PR on cozy-ui incoming.